src/cluster.c

```diff
@@ -89,8 +89,31 @@ void clusterRemoveNodeFromShard(clusterNode *node);
 int auxShardIdSetter(clusterNode *n, void *value, int length);
 sds auxShardIdGetter(clusterNode *n, sds s);
 int auxShardIdPresent(clusterNode *n);
+int auxTcpPortSetter(clusterNode *n, void *value, int length);
+sds auxTcpPortGetter(clusterNode *n, sds s);
+int auxTcpPortPresent(clusterNode *n);
+int auxTlsPortSetter(clusterNode *n, void *value, int length);
+sds auxTlsPortGetter(clusterNode *n, sds s);
+int auxTlsPortPresent(clusterNode *n);
 static void clusterBuildMessageHdr(clusterMsg *hdr, int type, size_t msglen);
 
+/* Inline functions for convenience. */
+inline int getNodeDefaultClientPort(clusterNode *n) {
```

Comment:
```suggestion
int getNodeDefaultClientPort(clusterNode *n) {
```
Presumably? The prototype in the header doesn't declare this inline, and unless it's static inline it usually isn't inlined anyways. I would move the top comment as well.